### PR TITLE
issue no:#2261 fix for xsrf panic error

### PR DESCRIPTION
**Pull request by sch00lb0y**
i don't have any idea why it not passing build. @astaxie pls help me with this

### CODE REVIEW

**Comment by astaxie** [comment]
@sch00lb0y Please fix the test case `TestErrorCode_01`

**Comment by sch00lb0y** [comment]
@astaxie I have fixed. Please review and let me know if need any changes.
Thanks

File: hooks.go [hunk 1]
```diff
@@ -22,16 +22,18 @@ func registerMime() error {
...
+		"403":         forbidden,
+		"404":         notFound,
+		"405":         methodNotAllowed,
+		"500":         internalServerError,
+		"501":         notImplemented,
+		"502":         badGateway,
+		"503":         serviceUnavailable,
+		"504":         gatewayTimeout,
+		"417":         invalidxsrf,
+		"422":         missingxsrf,
```
Comment (astaxie) [inline hooks.go]: do you have format this code? why there's a tab between the key and value

Comment (sch00lb0y) [inline hooks.go]: i have formatted now as you suggested.
Thanks

**Pull request merged**